randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1713,18 +1713,28 @@ Extra - asset_id: collision_camera_019
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
                   Bomb and Power Bombs
+              All of the following:
+                  # Power bombs (video: https://twitter.com/i/status/1545111396760690692 )
+                  Power Bombs ≥ 2
+              All of the following:
+                  # Bombs + flash shift (video: https://www.youtube.com/watch?v=NdVesJ6F_6A )
+                  Bomb and Flash Shift
```

Comment:
This again needs some sort of movement trick to be able to do things in time before all the blocks reform.